### ISSUE TITLE
Lack of diversity in research software

### ISSUE THREAD

**Issue by karthik** [si2-urssi/plan#86]
> Lack of diversity in research software

Most of this paragraph talks about lack of diversity in open source software. Could you say something here about lack of diversity in research software here? Perhaps the results of the US RSE survey?

**Comment by JeffCarver** [comment]
Yes, hopefully we can add a bit more once we finish the survey analysis. I will go back through what we have already to see if there is anything yet that we can say here.

**Comment by JeffCarver** [comment]
Demographics from the survey: 

We also asked participants to self-identify as male, female, other, or prefer not to say. The breakdown of gender identity was Male (n=734), Female (n=267) other (n=4) and prefer not to say (n=36) 

We also have something on age. We did not ask about ethnicity.

**Comment by karthik** [comment]
Jeff: Can you draft this as text that can go in the section?

**Comment by JeffCarver** [comment]
@karthik please see if this sentence is what you were asking for.

**Comment by danielskatz** [comment]
This seems very minimal.  We also have data from https://github.com/softwaresaved/international-survey/blob/master/analysis/2017/results_us_narrative.ipynb.

Maybe also add something like: 

The 164 US respondents to the 2018 International RSE Survey [@rse-survey] reported being 82% male, 14% female, and 4% preferred not to say. They also reported being 77% white, 11% Asian, 6% Hispanic/Latino, 5% other, and 2% Black. Additionally, 3% reported having a disability.

**Comment by danielskatz** [comment]
Actually, I'll add this to the PR.
